### PR TITLE
Web 139 remove moderator

### PR DESCRIPTION
**Pull request by razorman8669**

### CODE REVIEW

File: api/graphql/mutations/community.js [hunk 1]
```diff
@@ -12,9 +12,14 @@ export async function addModerator (userId, personId, communityId) {
   return community
 }
 
-export async function removeModerator (userId, personId, communityId) {
+export async function removeModerator (userId, personId, communityId, isRemoveFromCommunity) {
   const community = await getModeratedCommunity(userId, communityId)
-  await GroupMembership.removeModeratorRole(personId, community)
+  if (isRemoveFromCommunity) {
+    await CommunityService.removeMember(personId, communityId)
```
Comment (richchurcher) [inline api/graphql/mutations/community.js]: The only thing I wonder about here, and it's probably extra-picky, is that the moderator bit is effectively still set (even though the user is now `active = false` through being removed from the community). If the user is later re-added, oops, they're a moderator now! Would it be worth doing both, first zero the moderator role, then `removeMember`?

File: api/graphql/mutations/community.test.js [hunk 2]
```diff
@@ -52,11 +52,26 @@ describe('moderation', () => {
   })
 
   describe('removeModerator', () => {
-    it('works', async () => {
+    it('just removes moderator role', async () => {
```
Comment (richchurcher) [inline api/graphql/mutations/community.test.js]: For extra credit, you could add one that checks to ensure the mutation can't be used by a non-moderator :smile:

File: api/graphql/mutations/community.js (hunk 1 above)
Comment (richchurcher) [reply api/graphql/mutations/community.js]: Unless the user add code takes care of that, in which case we're fine. But I have a suspicion it might not.

Comment (razorman8669) [reply api/graphql/mutations/community.js]: nice catch.  I fixed as suggested (doing both)

**Pull request merged**